proto/stride/stakeibc/host_zone.proto

```diff
@@ -20,71 +20,114 @@ message CommunityPoolRebate {
   ];
 }
 
+// Core data structure to track liquid staking zones
```

Comment:
Honestly shocking we've gone this long without commenting this file

Thank you for commenting it so thoroughly!